### DISCUSSION TITLE
why frame by frame seeking backwards is slower than forward?

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#15305] [Q&A] [answered]
Hi, I use these keybinds `. frame-step` `, frame-back-step` and backward is very slow like 1 FPS, while forward is in realtime. Is there any solution to this?

Thank you :)

**Comment by hooke007** [comment, marked as answer]
https://github.com/mpv-player/mpv/issues/4019

**Reply by geextahslex** [reply]
Thank you, that helped :)
[Nucspl lua script](https://github.com/mpv-player/mpv/issues/4019#issuecomment-2469841936)

input.conf changes
mpv seek
```
. frame-step
#, frame-back-step
```

script seek
```
, script_binding frame-backstep
#. script_binding frame-forestep
```
